.maintain/gitlab/check_polkadot_companion_build.sh

```diff
@@ -12,7 +12,7 @@
 
 github_api_substrate_pull_url="https://api.github.com/repos/paritytech/substrate/pulls"
 # use github api v3 in order to access the data without authentication
-github_header="Accept: application/vnd.github.v3+json" 
```

Comment:
Is this Accept: header no longer needed?

Reply:
it's only needed to use github api v3 which allows unauthenticated requests. defaults to v4 then.